### PR TITLE
User creation form error feedback

### LINKED ISSUE #768: Missing visual feedback when user creation form is not complete
## Expected Behavior

I expect a visual feedback if I try to create an user without profil

## Current Behavior

The creation does not work and there is no visual feedback about what happen
missing-visual-feedback

## Possible Solution

Adds a red square around the missing inputs

## Steps to Reproduce
1. Start user creation and fill everything but the profile ID
2. Click on create
3.
4.

## Context (Environment)
Admin Console Version: next-console.kuzzle.io

### PR DESCRIPTION
**Pull request by berthieresteban**
## What does this PR do ?
fix #768 

### How should this be manually tested?
  - Step 1 : open AC -> security -> create user
  - Step 2 : try create without selected profile -> see error
  - Step 3 : 
  ...

### Other changes
b-alert component wasn't displayed, now show the catched error as in previous AC version

### Boyscout
/
### Screenshots (if appropriate)
/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "vue-router": "^3.1.3",
     "vue-template-compiler": "^2.6.10",
     "vue2-leaflet": "^1.0.2",
-    "vuejs-logger": "^1.5.3",
+    "vuejs-logger": "1.5.3",
```
Comment (Leodau) [inline package.json:64]: Why tho?

Comment (berthieresteban) [reply package.json:64]: Latest of vuejs-logger was bugged so I fixed to that version for now
cf: https://github.com/justinkames/vuejs-logger/issues/38

File: src/components/Security/Users/Steps/Basic.vue [hunk 2]
```diff
@@ -45,7 +46,8 @@ export default {
   },
   data() {
     return {
-      autoGenerateKuidValue: false
+      autoGenerateKuidValue: false,
+      profilesValid: null
```
Comment (Njuelle) [inline src/components/Security/Users/Steps/Basic.vue:50]: Nitpicking
```suggestion
      areProfilesValid: null
```
it's easier to know if it's a boolean

File: src/components/Security/Users/Steps/CredentialsSelector.vue [hunk 3]
```diff
@@ -73,11 +74,21 @@ export default {
     return {
       error: '',
       document: null,
-      id: null
+      id: null,
+      validLogin: null
```
Comment (Njuelle) [inline src/components/Security/Users/Steps/CredentialsSelector.vue:78]: same
```suggestion
      isLoginValid: null
```

File: src/components/Security/Users/Steps/UserProfileList.vue [hunk 4]
```diff
@@ -73,7 +74,8 @@ export default {
   data() {
     return {
       profileList: [],
-      selectedProfiled: 0
+      selectedProfiled: 0,
+      valid: null
```
Comment (Njuelle) [inline src/components/Security/Users/Steps/UserProfileList.vue:78]: same, `isValid`, also what is supposed to be valid?

**Pull request closed without merging**